TYPO3.Media/Classes/TYPO3/Media/Domain/Model/ThumbnailGenerator/FontDocumentThumbnailGenerator.php

```diff
@@ -75,7 +75,7 @@ public function refresh(Thumbnail $thumbnail)
             $red = imagecolorallocate($im, 0xFF, 0xFF, 0xFF);
             $black = imagecolorallocate($im, 0x00, 0x00, 0x00);
 
-            imagefilledrectangle($im, 0, 0, 1000, 1000, $red);
+            imagefilledrectangle($im, 0, 0, $width, $height, $red);
```

Comment:
But the resize below should take care to bring it down to the right size I think?